README.md

```diff
@@ -78,3 +37,53 @@ crontab -e
 ```
 
 *Note*: The time in adminVM is in UTC.
+
+## Special instructions
+
+### COXRAY
+
+The ETL is consist of two parts: `COXRAY_FILE` - for file upload and `COXRAY` for metadata submission.
```

Comment:
could you specify which one needs to run first and why

and also mention you need to wait for the ssjdispatcher to process the files before you can submit the metadata